src/components/DocumentationTopic/Summary/Availability.vue

```diff
@@ -10,18 +10,18 @@
 
 <template>
   <Section class="availability" role="complementary" :aria-label="$t('sections.availability')">
-    <Badge
+    <div
```

Comment:
Instead of rendering a `div` , it could be more semantic if we render `li` elements inside a `ul`.

Reply:
Maybe. I would definitely use a span or make sure to use inline or inline-block list items instead of a div block since these are all inline text that are meant to live on the same line(s).

Reply:
I changed it to use <span> for now.
I have a bug tracking doing a cleanup of the `Summary` component/folder, since we no longer really have a summary section. Maybe as a part of that I can look into treating availability info as a `ul`.

Reply:
Looking forward to that refactor of the Summary component! It's okay if we apply the semantic changes then.